src/recoil_values/__tests__/Recoil_atom-test.js

```diff
@@ -55,3 +55,19 @@ test('atom can store null and undefined', () => {
   act(() => set(myAtom, 'VALUE'));
   expect(get(myAtom)).toBe('VALUE');
 });
+
+test('atom can store a circular reference object', () => {
```

Comment:
Need to update the test to fix Flow errors, I can fix that when fixing the diff.  Let's see if our github sync scripts are working now....

Reply:
Interesting, Flow doesn't show any errors on my end. I guess it should be a missing property error on Circular class, there should be something wrong with my setup.